config/locales/en.yml

```diff
@@ -81,14 +81,64 @@ en:
         # API spec for Simple Server
         ## Sync APIs
         This API spec documents the endpoints that the devices (that are offline to varying levels) will use to sync data. The sync end points will send and receive bulk (a list of) entities. Both sending and receiving can be batched with configurable batch-sizes to accommodate low network bandwidth situations.
+
         ## Nesting resources
         The APIs have been designed to provide an optimal balance between accuracy and simplicity. Some of the APIs (patients) will be nested, and some other (blood pressures) will be flat.
-        ## Authentication
-        TODO
 
         ## Sync Mechanism
         Refer to the [related ADR](https://github.com/simpledotorg/simple-server/blob/master/doc/arch/001-synchronization.md).
 
+        ## API Authentication
+        There are two types of authentication available to access different parts of the Simple API.
+        - User Authentication - For medical professionals using the Simple App. Grants access to most of the Simple API
+          to read and write data for communities of patients.
+        - Patient authentication - For individual patients. Grants access to read a patient's own data.
+
+        ### User Authentication
+
+        A Simple client can make authenticated requests to the Simple API on behalf of a medical professional with the
```

Comment:
It sounds like all these creds are required .. can we be explicit about that? Maybe something like:

```
A client can make requests to the Simple API on behalf of a medical professional with the following three required credentials:
```